scripts/rollup/config.js

```diff
@@ -5,6 +5,12 @@ import replace from 'rollup-plugin-replace';
 import { uglify } from 'rollup-plugin-uglify';
 import filesize from 'rollup-plugin-filesize';
 
+if (process.env.SHIPJS === 'true' && !process.env.VERSION) {
```

Comment:
I can't imagine this ever matching again, we'd have to explicitly break the release script no?

Reply:
@Haroenv sorry I don't understand what you mean. Could you rephrase it?

Reply:
Since we added the env variable in shipjs configuration, and this check relies on the SHIPJS env variable, the chance that we ever get back into this branch is minimal IMO (much more likely to be forgotten in some other way. What about explicitly making version required, or asking for it interactively if not given (with the option to go for UNRELEASED)

Reply:
> What about explicitly making version required

I don't know what kind of stuff you have in mind,

but after your comment, I read the code again.

```js
if (process.env.SHIPJS === 'true' && !process.env.VERSION) {
```

That condition might not be understandable in a few months.
What if we just change it to

```js
if (process.env.NODE_ENV === 'production' && !process.env.VERSION) {
```

Ship.js prepends `NODE_ENV=production` and our previous release scripts did the same.
Only when we run `yarn build` on local machine, NODE_ENV is not `production`.
So it may be more explicit. What do you think?